app/models/monster.rb

```diff
@@ -0,0 +1,5 @@
+class Monster < ApplicationRecord
+  validates :name, presence: true
+  validates :source, presence: true
+  validates :img, presence: true
+end
```

Comment:
Nice validations, although I personally might refactor to `validates_presence_of :img, :name, :source`.